setup.cfg

```
@@ -30,7 +30,7 @@ tests_require = pytest-astropy
 setup_requires = setuptools_scm
 install_requires =
     numpy>=1.17
-    pyerfa>=1.7
+    pyerfa>=1.7,<1.7.2
```

Maybe this is safer in case we decided that the bug is really upstream and we can never use astropy with this particular bug fix version? (Not sure about the syntax itself.) 
```suggestion
    pyerfa>=1.7,!=1.7.2
```

---

Not sure which one is best, it could also be that tests on Astropy need to be updated. So we can update this when @mhvk can have a look.

---

Yes, definitely some tests in astropy need to be updated. But I don't understand the test that you first found to fail... Anyway, good to not have to worry about breaking unrelated builds!!!!

---

I think the one in PR is better, as we can't yet guarantee that we would be compatible with any of the future versions. In the past, there was only one version bundled, so it's probably not as a big deal to stay very restrictive within astropy.
(Ideally, we should be stating upper bounds on versions, too, hopefully, the package resolvers will get better support for that soonish)

---

Yes, agree that for now to just effectively pick a given version is best. I do hope to get around to fixing the astropy tests soon, and will try to make it version-dependent, as we do for numpy.